homeassistant/components/lovelace/__init__.py

```diff
@@ -249,3 +255,15 @@ def _register_panel(hass, url_path, mode, config, update):
         kwargs["sidebar_icon"] = config.get(CONF_ICON, DEFAULT_ICON)
 
     frontend.async_register_built_in_panel(hass, DOMAIN, **kwargs)
+
+
+async def _create_map_dashboard(hass: HomeAssistant):
+    dashboards_collection: dashboard.DashboardsCollection = hass.data[DOMAIN][
+        "dashboards_collection"
+    ]
+    await dashboards_collection.async_create_item(
+        {CONF_TITLE: "Map", CONF_URL_PATH: "map"}
```

Comment:
This is not great, we should have a way to set a translated title for the sidebar.